common/buildcraft/lib/gui/GuiBC8.java

```diff
@@ -57,6 +59,13 @@ public void drawGradientRect(int left, int top, int right, int bottom, int start
         return guiElements.stream().filter(elem -> elem.contains(x, y));
     }
 
+    protected void drawItemStackAt(ItemStack stack, int x, int y) {
+        RenderHelper.enableGUIStandardItemLighting();
+        RenderHelper.disableStandardItemLighting();
```

Comment:
It was mistake. I was should enable lighting for right block rendering, render item stack, and then disable.